src/zenml/zen_server/routers/workspaces_endpoints.py

```diff
@@ -484,98 +480,6 @@ def create_stack_component(
     return zen_store().create_stack_component(component=component)
 
 
-@router.get(
```

Comment:
Aren't those still workspace scoped and need an endpoint here?

Reply:
Flavors as a concept can't be workspace scoped, as such workspace_id needs to stay optional. However, for custom flavors we can still always ensure that they stay scoped.